packages/engine-test-utils/src/presets/engine-server/rename.ts

```diff
@@ -1599,7 +1599,7 @@ const NOTES = {
           expected: "foo1",
         },
         {
-          actual: changedEntries && changedEntries.length === 5,
+          actual: changedEntries && changedEntries.length === 6,
```

Comment:
curiosu - why did this change?

Reply:
Previously we were missing the changeEntry where root.children got updated from "[foo]" to "[]" before foo got deleted.

In the diff, I added a couple places where we forgot to propagate all the changedEntries.